hugo/content/playground/common.ts

```diff
@@ -220,10 +220,10 @@ async function getFreshDSLWrapper(
     languageId,
     code,
     worker,
-    monarchGrammar: generateMonarch(Grammar, languageId)
+    textmateGrammar: generateTextMate(Grammar, {id: languageId, grammar: 'UserGrammar'})
   }), htmlElement).then(() => {
```

Comment:
We should use `await` and wrap everything in a big try, then you can just return the `wrapper` and `undefined` then there is no need for `as ...` at the end if I am not mistaken.

Reply:
This will be changed with #222 as the function changed anyway.